src/rebar_relx.erl

```diff
@@ -195,6 +199,32 @@ merge_overlays(Config) ->
     NewOverlay = lists:flatmap(fun({overlay, Overlay}) -> Overlay end, lists:reverse(Overlays)),
     [{overlay, NewOverlay} | Others].
 
+overlay_vars(Opts) ->
+    case proplists:get_value(overlay_vars, Opts) of
+        undefined ->
+            [];
+        OverlayVars ->
+            [OverlayVars]
+    end.
+
+maybe_obey_command_args(RelxConfig, Opts, Args) ->
+    lists:foldl(
+        fun({Opt, AliasOpt}, Acc) ->
+            case proplists:get_value(Opt, Opts) of
+                undefined ->
+                    Acc;
+                V ->
+                    lists:keystore(AliasOpt, 1, Acc, {AliasOpt, V})
+            end;
+        (Opt, Acc) ->
```

Comment:
Formatting, just need to indent this and the lines below it to match the args of the fun above it.